### PR TITLE
feat(tooltip): render children when disabled or text is missing

### PR DESCRIPTION
**Pull request by priyajeet**
The tooltip will not show and just render its children when `text` is missing or if `isDisabled` is `true`.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @priyajeet has signed the CLA. Thanks for the pull request!

File: src/components/tooltip/__tests__/Tooltip-test.js [hunk 1]
```diff
@@ -203,6 +203,22 @@ describe('components/tooltip/Tooltip', () => {
 
             expect(wrapper.find('[role="tooltip"]').hasClass('is-error')).toBe(true);
         });
+
+        test('should render children only when tooltip is disabled', () => {
```
Comment (priyajeet) [inline src/components/tooltip/__tests__/Tooltip-test.js]: bogus

**Comment by jeremypress** [comment]
`src/elements/common/Tooltip.js` We added this before the merger which does something similar if you want to re-use.

**Comment by priyajeet** [comment]
> `src/elements/common/Tooltip.js` We added this before the merger which does something similar if you want to re-use.

Plan is to nuke that

**Pull request merged**